sdccc/src/test/java/com/draeger/medical/sdccc/util/MdibBuilder.java

```diff
@@ -435,6 +438,13 @@ public Pair<NumericMetricDescriptor, NumericMetricState> buildNumericMetric(
         return new ImmutablePair<>(numericMetricDescriptor, numericMetricState);
     }
 
+    /**
+     * @return a new range instance.
+     */
+    public Range buildRange() {
+        return participantModelFactory.createRange();
```

Comment:
This doesn't really add anything and should be removed.

Reply:
done